pydis_site/apps/resources/resources/software_design_by_example.yaml

```diff
@@ -0,0 +1,21 @@
+name: Software Design by Example
+description: A tool-based introduction to Software Design with Python. This book teaches
+  design by explaining examples of small versions of familiar tools to show how
+  experienced software designers think. It introduces some fundamental ideas in computer
+  science that many self-taught programmers haven’t encountered.
+title_image: https://third-bit.com/sdxpy/logo.svg
+title_url: https://third-bit.com/sdxpy/
+urls:
+- icon: branding/github
+  url: https://github.com/gvwilson/sdxpy
+  color: black
+tags:
+  topics:
+    - general
+    - software design
+  payment_tiers:
+    - free
```

Comment:
I think we _could_ set this as both free and paid - since you can read it online for free (or, in April next year?) buy it. We do the same for "A Byte of Python" which has the same model.

Reply:
Good catch, done.